manifests/config.pp

```diff
@@ -22,4 +22,16 @@
     mode   => '0750',
   }
 
+  foreman_config_entry { 'pulp_client_cert':
+    value          => $client_cert,
+    ignore_missing => false,
+    before => Exec['foreman-rake-db:migrate'],
+  }
+
+  foreman_config_entry { 'pulp_client_key':
+    value          => $client_key,
+    ignore_missing => false,
+    before => Exec['foreman-rake-db:migrate'],
+  }
```

Comment:
@ehelms  was actually suggesting it go here:  https://github.com/Katello/puppet-katello/blob/master/manifests/config.pp

I'll let him comment if he cares one way or the other